README.md

```diff
@@ -93,10 +71,42 @@ getPackageName ('package.json')
 
 ### Global Bundle (CDN)
 
-Fluture is hosted in full with all of its dependencies at
-https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+To load Fluture directly into a browser, a code pen, or [Deno][], use one of
+the following downloads from JSDelivr. They are single files that come with all
+of Fluture's dependencies pre-bundled.
+
+- [Fluture Script][]: A JavaScript file that adds `Fluture` to the global scope.
+- [Fluture Script Minified][]: The same as above, but minified.
+- [Fluture Module][]: An EcmaScript module with named exports.
+- [Fluture Module Minified][]: The same as above, but minified.
 
-This script will add `Fluture` to the global scope.
+[Fluture Script]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+[Fluture Script Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.min.js
+[Fluture Module]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.mjs
+[Fluture Module Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.min.mjs
```

Comment:
I have pushed the updated bundles to a separate branch `dist` for testing the various scripts as they will be picked up by jsdelivr. See:

- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/bundle.js
- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/bundle.min.js
- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/bundle.mjs
- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/bundle.min.mjs

Reply:
Hm. I see the mjs file is not available in minified format. I will try again with a different file name.

Reply:
It does work with the new names:

- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/module.js
- https://cdn.jsdelivr.net/gh/fluture-js/Fluture@dist/dist/module.min.js